index.html

```diff
@@ -4683,82 +4683,134 @@ <h2>Internationalization Considerations</h2>
     </p>
 
     <section class="informative">
-      <h3>Language Declarations</h3>
+      <h3>Language and Direction</h3>
 
       <p>
-When expressing human-readable text content in the data model, it is important
-for presentation, accessibility, and further processing to specify the language
-of the text, as follows:
+Data publishers are strongly encouraged to use the design pattern described
```

Comment:
It's hard to get excited for non-normative language such as "strongly encouraged". Is that like *SHOULD*?

Reply:
Unfortunately, we're in the W3C Candidate Recommendation phase and don't have enough time in our charter to go through another CR phase (our charter would expire), so we're language lawyering to provide strong guidance w/o triggering another CR. It's the best we can do right now. I expect the next revision of the specification will convert parts of this section into normative language.

Reply:
I think this is a false urgency. It's easier than before to move back through CR than in older versions of the process and artificial dates (including charter expirations) should not gate doing the right thing. Charters have been extended for short periods of time in the past. This is particularly the case here, where the normative language is in what is effectively "advice to authors" and doesn't impact implementations.

Reply:
We have already received an administrative charter extension. We have done the math and quite literally cannot go through another CR w/o expiring our charter. The WG has already had this discussion. We can talk more about this on Wednesday if you'd like.

Reply:
We covered this in the call, and it doesn't matter, we're trying to do the right thing and are working toward that solution with the i18n community. Resolving this conversation as it's not useful wrt. the latest PR.